### PR TITLE
new monitoring pkg: servicemonitor handler added

### PR DESCRIPTION
**Pull request by elenagerman**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -28,14 +28,15 @@ require (
 	github.com/openshift/ptp-operator v0.0.0-20240404165119-29a3d7b3d60b
 	github.com/operator-framework/api v0.23.0
 	github.com/operator-framework/operator-lifecycle-manager v0.28.0
+	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.72.0
```
Comment (kononovn) [inline go.mod:31]: This version is too new for eco-goinfra because it based on api v0.30.1 Please use v0.73.2 instead

**Review by kononovn — CHANGES_REQUESTED** [review]
Please change version and reinstall dependency

File: go.mod (hunk 1 above)
Comment (elenagerman) [reply go.mod:31]: @kononovn sorry, I did not understand your request: v.72.0 is older then  v0.73.2, no?

Comment (elenagerman) [reply go.mod]: done, but I still do not understand your point

**Comment by elenagerman** [comment]
@kononovn @trewest @cdvultur @achuzhoy PTAL

**Review by kononovn — CHANGES_REQUESTED** [review]
Just 1 cosmetic change.

**Review by kononovn — APPROVED** [review]
/lgtm

**Comment by elenagerman** [comment]
@trewest PTAL

**Review by cdvultur — APPROVED** [review]
/lgtm

was about to write about conflict, but I see that is sorted out

**Review by trewest — APPROVED** [review]
/lgtm

**Pull request merged**